versions/3.0.4.md

```diff
@@ -2152,7 +2153,7 @@ Because examples using these fields represent the final serialized form of the d
 The singular `example` field in the Parameter or Media Type Object is concise and convenient for simple examples, but does not offer any other advantages over using Example Objects under `examples`.
 
 Some examples cannot be represented directly in JSON or YAML.
-For all three ways of providing examples, these can be shown as string values with any escaping necessary to make the string valid in the JSON or YAML format of the OpenAPI Description document.
+For all three ways of providing examples, these can be shown as string values with any escaping necessary to make the string valid in the JSON or YAML format of the OpenAPI Description.
```

Comment:
An OpenAPI Description could plausibly be in a mixture of JSON and YAML documents, so this ought to be "the JSON or YAML format of the OpenAPI Document."